Source/WebGPU/WebGPU/Queue.mm

```diff
@@ -495,8 +495,8 @@ static void invalidateCommandBuffers(Vector<std::reference_wrapper<CommandBuffer
     if (!device)
         return;
 
-    auto& texture = fromAPI(destination.texture);
-    if (texture.isDestroyed()) {
+    Ref texture = Ref { fromAPI(destination.texture) };
```

Comment:
protectedFromAPI() would be slightly more style consistent here.

Reply:
More conventional code is
```
Ref texture = fromAPI(...);
```